radixdlt-core/radixdlt/src/main/java/org/radix/Radix.java

```diff
@@ -17,33 +17,32 @@
 
 package org.radix;
 
+import org.apache.commons.cli.ParseException;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.bouncycastle.jce.provider.BouncyCastleProvider;
+import org.json.JSONObject;
+import org.radix.api.http.RadixHttpServer;
+import org.radix.utils.IOUtils;
+
 import com.google.common.collect.ImmutableMap;
 import com.google.inject.Guice;
-import com.google.inject.Injector;
 import com.google.inject.Key;
 import com.google.inject.TypeLiteral;
-import com.radixdlt.RadixNodeModule;
-import com.radixdlt.consensus.bft.Self;
-import com.radixdlt.utils.MemoryLeakDetector;
 import com.radixdlt.ModuleRunner;
+import com.radixdlt.RadixNodeModule;
 import com.radixdlt.consensus.bft.BFTNode;
-import com.radixdlt.systeminfo.SystemInfoRunner;
+import com.radixdlt.consensus.bft.Self;
 import com.radixdlt.network.addressbook.PeerManager;
 import com.radixdlt.properties.RuntimeProperties;
-
-import java.util.Map;
-import org.apache.logging.log4j.LogManager;
-import org.apache.logging.log4j.Logger;
-import org.apache.commons.cli.ParseException;
-import org.bouncycastle.jce.provider.BouncyCastleProvider;
-import org.json.JSONObject;
-import org.radix.api.http.RadixHttpServer;
-import org.radix.utils.IOUtils;
+import com.radixdlt.systeminfo.SystemInfoRunner;
+import com.radixdlt.utils.MemoryLeakDetector;
 
 import java.io.IOException;
 import java.io.InputStream;
 import java.net.URISyntaxException;
 import java.security.Security;
+import java.util.Map;
 import java.util.Properties;
 
 public final class Radix {
```

Comment:
Doesn't seem like there are any functional changes here. Would be better to not include cosmetic-only changes.

Reply:
Rolled back changes. For these minor changes I'll create a separate PR.